### PR TITLE
FISH-7433: upgrading metrics version

### PR DESCRIPTION
**Pull request by breakponchito**
This is to patch metrics version 5.1.0-RC1 and add changes to skip OSGI issue regarding java packages

### CODE REVIEW

File: api/pom.xml [hunk 1]
```diff
@@ -19,13 +19,17 @@
...
-        <version>5.1.0-RC1</version>
+        <version>5.1.0-RC1.payara-p1</version>
     </parent>
 
     <artifactId>microprofile-metrics-api</artifactId>
     <name>MicroProfile Metrics API</name>
     <description>MicroProfile Metrics :: API</description>
 
+    <properties>
+        <bnd.baseline.continue.on.error>true</bnd.baseline.continue.on.error>
```
Comment (aubi) [inline api/pom.xml:30]: Was this used only for development or is it still necessary?

Comment (Pandrex247) [reply api/pom.xml:30]: It's to bypass the BND tool complaining about the version number "not changing" despite there being changes - it doesn't understand our patch qualifier.

**Review by Pandrex247 — COMMENTED** [review]
It might be worth skipping this. 
5.1.0 (final) is out now: https://github.com/eclipse/microprofile-metrics/tree/5.1.0

**Review by luiseufrasio — APPROVED** [review]
LGTM

**Pull request merged**